app/services/github_service.rb

```diff
@@ -28,8 +28,8 @@ def load_repositories(access_token)
       # Add repos and remove user from any repos they no longer have access to
       remove_outdated(user, outdated_repos)
     end
-
-    def submit_issue(repo_id, sub_name, email, title, details)
+    
+    def submit_issue(repo_id, sub_name, email, email_public, title, details)
```

Comment:
You should just need to add `# rubocop:disable Metrics/ParameterLists` before the definition, `# rubocop:enable Metrics/ParameterLists` after, and then also remove the trailing whitespace on line 31 to satisfy Rubocop.